code401challenges/src/main/java/code401challenges/utilities/MultiBracketValidation.java

```diff
@@ -0,0 +1,43 @@
+package code401challenges.utilities;
+
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Stack;
+
+public class MultiBracketValidation {
+
+    public static Stack<Character> stack;
+
+
+
+    public static boolean multiBracketValidation(String input){
+        if(input.length() == 1)
+            return false;
+
```

Comment:
Need to instantiate your stack;
```suggestion
stack = new Stack<>();
```